common/resource/fx.go

```diff
@@ -339,6 +352,7 @@ func RPCFactoryProvider(
 	resolver *membership.GRPCResolver,
 	tracingStatsHandler telemetry.ClientStatsHandler,
 	monitor membership.Monitor,
+	dynamicConfig *configs.Config,
```

Comment:
why do we need history service configs here? this rpc factory is for all services